### PR TITLE
Add first version of Freshdesk's feedback widget

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

**Comment by irlaec** [comment]
[Freshdesk feedback integration](https://trello.com/c/VlSgejlk/96-freshdesk-feedback-integration)

**Comment by irlaec** [comment]
Closes #453

File: octoprint_mrbeam/static/js/loadingoverlay_viewmodel.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ $(function() {
             if (!event.target.activeElement.href) {
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/loadingoverlay_viewmodel.js:13]: Line 12: Please do not use `window.onbeforeunload = function` use `$( window ).on('beforeunload', function(){`

File: octoprint_mrbeam/static/js/feedback_widget.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            if(MRBEAM_SW_TIER === 'BETA') {
+                channel = 'Beta'
+            } else {
+                channel = 'Stable'
+            }
+
+            try{
+                window.FreshWidget.init("", {
+                    "queryString": "&widgetType=popup"
+                        + "&helpdesk_ticket[custom_field][cf_serial_922577]=" + MRBEAM_SERIAL
```
Comment (andywerner) [inline octoprint_mrbeam/static/js/feedback_widget.js:23]: Please also pre-fill the users email address but make it editable for the user

**Review by andywerner — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/static/js/loadingoverlay_viewmodel.js (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/static/js/loadingoverlay_viewmodel.js:13]: changed!

**Review by andywerner — APPROVED** [review]

**Pull request merged**